android/KMEA/app/src/main/AndroidManifest.xml

```diff
@@ -13,7 +13,6 @@
 
         <activity
             android:name="com.tavultesoft.kmea.KeyboardPickerActivity"
-            android:launchMode="singleTask"
```

Comment:
This fixes the Keyboard picker menu from getting separated from the Keyman app / keyboard

Reply:
I am a little wary about this change; what other potential impacts are there?

Reply:
This line got added during 14.0 in PR #4531 for improving the welcome.htm responsiveness.
I'll do some testing removing the line impacts viewing help docs.

Reply:
So has this broken something in 14.0 with the keyboard picker menu?

Reply:
Hm, I reverted this change and the keyboard picker seems fine.